TACC/Core-CMS-Custom
# View Registrations table - remove modal link

## Overview
Removed modal link
…

## Related

…

## Changes
Commented out <a> element which opens modals
…

## Testing

1. Go to http://localhost:8000/administration/list-registration-requests/
2. Confirm UI looks like below

## UI
![image](https://user-images.githubusercontent.com/43251554/194420904-c9a13063-da6b-43da-9d3f-f92b0197c7ce.png)

…
